lib/fluent/daemon.rb

```diff
@@ -9,7 +9,5 @@
 
 server_module = Fluent.const_get(ARGV[0])
 worker_module = Fluent.const_get(ARGV[1])
-# it doesn't call ARGV in block because when reloading config, params will be initialized and then it can't use previous config.
-config_path = ARGV[2]
-params = JSON.parse(ARGV[3])
-ServerEngine::Daemon.run_server(server_module, worker_module) { Fluent::Supervisor.load_config(config_path, params) }
+params = JSON.parse(ARGV[2])
```

Comment:
Memo: daemon.rb is used here and original `ARGV[2]`(`config_path`) is removed in this commit:
https://github.com/fluent/fluentd/blob/041af39f401b26002c07a299ad7c85346909ce7c/lib/fluent/supervisor.rb#L454-L458

Reply:
Yes, sorry for the lack of explanation.